kolibri/core/assets/src/vue/content-renderer/download-button.vue

```diff
@@ -6,15 +6,15 @@
       class="dropdown-button"
       @click="toggleDropdown"
       aria-haspopup="true">
-      <span class="dropdown-button-text" :class="{uparrow: dropdownopen}">
+      <span class="dropdown-button-text" :class="{'dropdown-button-text-open': dropdownOpen}">
         {{ $tr('downloadContent') }}
       </span>
     </button>
     <ul
       ref="dropdownitems"
       class="dropdown-items"
-      :class="{ dropdownopen: dropdownopen }"
-      :aria-hidden="dropDownOpenText"
+      :class="{ 'dropdown-open': dropdownOpen }"
+      :aria-hidden="dropdownOpenText"
```

Comment:
The `aria-hidden` attribute is currently being set to the string `'true'` when the dropdown is open, which is the opposite of the intended behavior.

Perhaps all of this `display: none` and `aria-hidden` logic can be replaced with a simple `hidden` attribute?

* https://gist.github.com/indirectlylit/fcd479c06ee2738a7ac33e0a94446af2#file-download-button-vue-L17
* https://www.paciellogroup.com/blog/2013/11/html5-accessibility-chops-hidden-aria-hidden-support/